release/Earthfile

```diff
@@ -26,6 +26,8 @@ release-github:
         ../+earth-all/* ./release/
     ARG BODY="No details provided"
     RUN --secret GITHUB_TOKEN=+secrets/GITHUB_TOKEN test -n "$GITHUB_TOKEN"
+    RUN ls -la ./release
+    RUN asdf
```

Comment:
Unintended?

Reply:
oh yikes! `git commit -a` failed me. will fix in master.

Reply:
fixed in https://github.com/earthly/earthly/commit/1f9ca76b97a37a1524a12dc97ceb5b386e911ce6 sorry about that slip up.